src/resources/lang/en/en.json

```diff
@@ -954,7 +954,20 @@
   "payment_message_button": "Thank you for your payment of :amount.",
   "payment_type_direct_debit": "Direct Debit",
   "bank_accounts": "Credit Cards & Banks",
+  "transactions": "Transactions",
+  "new_transaction": "New Transaction",
+  "created_transaction": "Successfully Created Transaction",
+  "deleted_transaction": "Successfully Deleted Transaction",
+  "updated_transaction": "Successfully Updated Transaction",
+  "edit_transaction": "Edit Transaction",
+  "delete_transaction": "Delete Transaction",
+  "archive_transaction": "Archive Transaction",
   "add_bank_account": "Add Bank Account",
+  "transaction_details": "Transaction Details",
+  "deposit": "Deposit",
+  "withdrawal": "Withdrawal",
+  "unmatched": "Unmatched",
+  "matched": "Matched",
```

Comment:
Please never edit `en.json` or any translation files manually. They are automatically generated in `invoiceninja/invoiceninja` repo.

If you're missing translation reach out to David or me.

Reply:
> Please never edit `en.json` or any translation files manually. They are automatically generated in `invoiceninja/invoiceninja` repo.
> 
> If you're missing translation reach out to David or me.

removed